packages/core/src/bar-chart.ts

```diff
@@ -193,7 +193,7 @@ export class BarChart extends BaseAxisChart {
 		this.dispatchEvent("update");
 	}
 
-	updateElements(animate: boolean, rect?: any, g?: any) {
+	updateElements(animate: boolean, rect?: any) {
```

Comment:
Is there a reason to keep `rect?: any`?

Reply:
Potentially we could but I don't think it's necessary. The only time the rect parameter is used is when new data is bound to the bars, and a reference to this is passed into UpdateElements. So potentially, we could move this into UpdateElement, but then later on in the code, the same reference is used to define an exit transition, which we would also have to move into UpdateElement unless we want to keep 2 references to the bars with newly bound data.
`		const rect = g.selectAll("rect.bar")
			.data((d, index) => this.addLabelsToDataPoints(d, index));

		this.updateElements(true, rect);
                ....
		rect.exit()  // We would either have to move this inside UpdateElement as well
                                 // or keep two references to rect
			.transition(this.getDefaultTransition())
			.style("opacity", 0)
			.remove();
`

Reply:
ok, if we do sometimes pass in things for rect then we can keep it, but you don't need to pass in `null` for it, since it's an optional parameter